sql/sqlserver/server.go

```diff
@@ -674,22 +676,25 @@ func (s *Server) processColumns(desc *structured.TableDescriptor,
 
 func (s *Server) processInsertRows(node parser.SelectStatement) (rows []sqlwire.Result_Row, err error) {
 	switch nt := node.(type) {
+	// case *parser.Select:
+	// case *parser.Union:
+	// TODO(vivek): return s.query(nt.stmt, nil)
 	case parser.Values:
 		for _, row := range nt {
 			var vals []sqlwire.Datum
 			for _, val := range row {
 				switch vt := val.(type) {
 				case parser.StrVal:
-					tmp := string(vt)
-					vals = append(vals, sqlwire.Datum{StringVal: &tmp})
-				case parser.NumVal:
-					tmp := string(vt)
-					vals = append(vals, sqlwire.Datum{StringVal: &tmp})
-				case parser.ValArg:
-					return rows, util.Errorf("TODO(pmattis): unsupported node: %T", val)
+					vals = append(vals, sqlwire.Datum{StringVal: (*string)(&vt)})
```

Comment:
Add a TODO that we actually need to be calling `parser.EvalExpr` on each row. This is necessary so that we can handle insertions like:

```
  INSERT INTO foo VALUES (1+1)
```

The result from `EvalExpr` will be a `Datum` that we can type assert to `DTuple`. This is where conversion of `NumVal` into int/float will take place.